class-two-factor-core.php

```diff
@@ -1721,12 +1721,10 @@ public static function user_two_factor_options( $user ) {
 
 		wp_nonce_field( 'user_two_factor_options', '_nonce_user_two_factor_options', false );
 		?>
+		<h3><?php esc_html_e( 'Two-Factor Options', 'two-factor' ); ?></h3>
 		<input type="hidden" name="<?php echo esc_attr( self::ENABLED_PROVIDERS_USER_META_KEY ); ?>[]" value="<?php /* Dummy input so $_POST value is passed when no providers are enabled. */ ?>" />
 		<table class="form-table">
 			<tr>
-				<th>
-					<?php esc_html_e( 'Two-Factor Options', 'two-factor' ); ?>
-				</th>
 				<td>
```

Comment:
Is it possible to remove the wrapping `<table>`? Does it make any sense to do that?

Looking at application passwords, that uses a `div.application-passwords-list-table-wrapper` here.

That might remove the need for the CSS change as well, as `table.form-table` would no longer be applying.

Reply:
I've removed the `table. form-table` it was adding some CSS to make it 100% width, so have added CSS for the existing `two-factor-methods-table` table.